workflow-job

```diff
@@ -719,7 +719,8 @@ def readBlobStoreImplementation() {
 def startObjectStore() {
     switch (readBlobStoreImplementation()) {
         case "s3":
-            sh "docker run --detach=true --name=${containers.s3} ${images.s3}"
+            s3_envs = "--env SCALITY_ACCESS_KEY_ID=accessKey1 --env SCALITY_SECRET_ACCESS_KEY=secretKey1 --env S3BACKEND=mem"
+            sh "docker run --detach=true --name=${containers.s3} ${s3_envs} ${images.s3}"
```

Comment:
what is the value of conainers.s3? For your change to work, you have to set the name of the image as `s3.docker.test`, or to define a network alias, or to provide a config.json file.

Reply:
The container name is defined here:
https://github.com/linagora/james-jenkins/blob/7e20884a4533c938c1672e18091e97f566707479/workflow-job#L227

= 's3-' + buildId

The image name is defined here:
https://github.com/linagora/james-jenkins/blob/7e20884a4533c938c1672e18091e97f566707479/workflow-job#L172

= properties."s3" ?: 'zenko/cloudserver:8.2.6'

Reply:
Let's do this via the link